app/src/main/java/com/infomaniak/drive/ui/menu/MenuPicturesFragment.kt

```diff
@@ -0,0 +1,68 @@
+/*
+ * Infomaniak kDrive - Android
+ * Copyright (C) 2021 Infomaniak Network SA
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.infomaniak.drive.ui.menu
+
+import android.os.Bundle
+import android.os.CountDownTimer
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.view.ViewCompat
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import com.infomaniak.drive.R
+import com.infomaniak.lib.core.utils.Utils.createRefreshTimer
+import kotlinx.android.synthetic.main.fragment_menu_pictures.*
+
+class MenuPicturesFragment : Fragment() {
+
+    private val timer: CountDownTimer by lazy {
+        createRefreshTimer { swipeRefreshLayout?.isRefreshing = true }
+    }
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        return inflater.inflate(R.layout.fragment_menu_pictures, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        toolbar.setNavigationOnClickListener {
+            findNavController().popBackStack()
+        }
+
+        ViewCompat.requestApplyInsets(pictureListCoordinator)
+
+        val picturesFragment = PicturesFragment {
+            timer.cancel()
+            swipeRefreshLayout.isRefreshing = false
+        }
+
+        swipeRefreshLayout.setOnRefreshListener {
+            picturesFragment.reloadPictures()
+        }
+
+        timer.start()
+        if (childFragmentManager.findFragmentByTag("TOTO") == null) {
+            childFragmentManager.beginTransaction()
+                .replace(R.id.picturesFragmentView, picturesFragment, "TOTO")
```

Comment:
TOTO I think it's an oversight  😂, `PicturesFragment.TAG` it'll be a good idea i think